nats-base-client/types.ts

```diff
@@ -516,6 +516,11 @@ export interface StoredMsg {
   time: Date;
 }
 
+export interface DirectMsg extends StoredMsg {
+  stream: string;
```

Comment:
This public? Should just look like a message you can get today with kv.Get()

Reply:
No, it is not, they see exactly the same API